### PR TITLE
Initial addition of logo/image

### PR DESCRIPTION
**Pull request by SQLDBAWithABeard**
So this is the first attempt at adding the logo to the web-site

I have also added some super quick instructions on how to run this locally so you can see it in a browser on your machine. This will require WSL2 and docker. Full instructions here

https://sqlcollaborative.github.io/datasaturdays-blog/blog/community/data%20saturdays/2021/04/15/locally-viewing-github-pages-new-data-saturdays.html

So what has been changed?

On the front page, the image, the favicon and a link to the blog

image

for future events - the icon

image

for past events - a different icon so you can see which you like - I like a differentiation but have no problem with past and future being the same

image

for events - I have added the default logo and the favicon

image

Thoughts/comments

### CODE REVIEW

**Review by rabryst — COMMENTED** [review]
How large is the web safe 256-colour PNG version compared to JPEG? I don't like lossy images for logos.

**Comment by SQLDBAWithABeard** [comment]
Which one @rabryst  ? I used the c.facebook%201r.png just to have something to look at

File: index.html [hunk 1]
```diff
@@ -89,7 +90,7 @@ <h2 class="section-heading text-uppercase">Past Events</h2>
             <td>
               <div class="hero-box mt-3 align-self-center">
                 <a href="{{ post.url | absolute_url }}">
-                  <img src="assets/img/logos/datasaturday_temp.png" height="65" />
+                  <img src="assets/design/logo/11.jpg" height="65" />
```
Comment (rabryst) [inline index.html:93]: This one, @SQLDBAWithABeard

Comment (SQLDBAWithABeard) [reply index.html:93]: gotcha, I was tired. looks like this

image

**Review by spaghettidba — APPROVED** [review]

**Pull request merged**